persistence/src/main/java/edu/unc/lib/dl/persist/services/delete/MarkForDeletionJob.java

```diff
@@ -44,9 +44,8 @@
     private AgentPrincipals agent;
     private PID pid;
 
-    public MarkForDeletionJob(PID pid, AgentPrincipals agent,
-            RepositoryObjectLoader repositoryObjectLoader, SparqlUpdateService sparqlUpdateService,
-            AccessControlService aclService) {
+    public MarkForDeletionJob(PID pid, AgentPrincipals agent, RepositoryObjectLoader repositoryObjectLoader,
```

Comment:
this is okay, but in general it's safest to avoid formatting changes in unchanged files since those can tend to result in merge conflicts

Reply:
this file had other changes, before i realized they needed to be in the service class instead. this must just have been left over. i can remove it if you'd like. there was an extra param in the constructor for the message sender, which is why the formatting got changed